tasks.py

```diff
@@ -5,48 +5,59 @@
 
 # Workaround for homebrew installation of Python (https://bugs.python.org/issue22490)
 import os
-os.environ.pop('__PYVENV_LAUNCHER__', None)
+os.environ.pop("__PYVENV_LAUNCHER__", None)
```

Comment:
we were mixing `'` and `"` so a bunch of these changes are just cosmetic

Reply:
Thanks!